android-client-sdk/src/test/java/com/devcycle/sdk/android/api/DVCLifecycleCallbacksTests.kt

```diff
@@ -0,0 +1,55 @@
+package com.devcycle.sdk.android.api
+
+import android.app.Activity
+import android.os.Handler
+import com.devcycle.sdk.android.eventsource.DVCLifecycleCallbacks
+import com.devcycle.sdk.android.eventsource.EventSource
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.*
+import org.mockito.stubbing.Answer
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.TimeUnit
+
+class DVCLifecycleCallbacksTests {
+
+    private val mockEventSource: EventSource = mock(EventSource::class.java)
+    private val mockActivity: Activity = mock(Activity::class.java)
+    private val mockHandler: Handler = mock(Handler::class.java)
+    private var countDownLatch = CountDownLatch(1)
+
+
+    @BeforeEach
+    fun setup() {
+        countDownLatch = CountDownLatch(1)
+
+        val mockPostDelayed = Answer<Void?> { invocation ->
+            val runnable: Runnable = invocation.arguments[0] as Runnable
+            Thread.sleep(invocation.arguments[1] as Long)
+            runnable.run()
+            countDownLatch.countDown()
+            null
```

Comment:
what's this null doing?

Reply:
its the return type of the lambda, i don't fully get the point of it but when it's not there it errors out because the lambda function is returning the wrong type